go.mod

```diff
@@ -6,14 +6,14 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo/v2 v2.6.1
 	github.com/onsi/gomega v1.24.1
-	github.com/operator-framework/api v0.14.0
+	github.com/operator-framework/api v0.17.4-0.20230223191600-0131a6301e42
 	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/client_model v0.3.0
-	k8s.io/api v0.26.0
-	k8s.io/apimachinery v0.26.0
-	k8s.io/client-go v0.26.0
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
-	sigs.k8s.io/controller-runtime v0.14.1
+	k8s.io/api v0.26.1
+	k8s.io/apimachinery v0.26.1
+	k8s.io/client-go v0.26.1
+	k8s.io/utils v0.0.0-20230220204549-a5ecb0141aa5
+	sigs.k8s.io/controller-runtime v0.14.4
```

Comment:
could we make this 0.14.5, since we are on it. The patch release reverts a few changes which may not necessarily affect here directly, but its better to use the latest patch with all the changes.

Reply:
Yup! Bumped controller-runtime to `v0.14.5` as well